README.md

```diff
@@ -1 +1,91 @@
-# friendly-giggle
+# pet-cam-fg
+Pet cam client and API that serves connection to user and their pet. This project is intended to run alongside [pet-cam-aa](https://github.com/wongbros/automatic-adventure).

```

Comment:
Change first sentence:

Pet cam User client that identifies user and establishes connection to video feed.

Or something along that.